app/views/defence_requests/form_partials/_case_details.html.erb

```diff
@@ -28,16 +28,17 @@
 </div>
 
 <div class="form-group">
-  <%= f.label :investigating_officer_name, label_text_for_form(attribute_name: 'investigating_officer_name'), class: "form-label-bold" %>
+  <%= f.label :investigating_officer_name, label_text_for_form(attribute_name: 'investigating_officer_name', optional: true), class: "form-label-bold" %>
   <%= f.text_field :investigating_officer_name, label: t('investigating_officer_name'), class: 'text-field custody-number' %>
 </div>
 
 <div class="form-group">
-  <%= f.label :investigating_officer_contact_number, label_text_for_form(attribute_name: 'investigating_officer_contact_number'), class: "form-label-bold" %>
+  <%= f.label :investigating_officer_contact_number, label_text_for_form(attribute_name: 'investigating_officer_contact_number', optional: true), class: "form-label-bold" %>
   <%= f.text_field :investigating_officer_contact_number, label: t('investigating_officer_contact_number'), class: 'text-field custody-number' %>
 </div>
 
 <div class="form-group">
   <%= f.label :comments, label_text_for_form(attribute_name: "comments", optional: true), class: "form-label-bold" %>
+  <aside><%= t("comments_field_aside_text") %></aside>
```

Comment:
From accessibility perspective, the `<aside>` tag should be inside the label as well. With the form builder you can do 

```ruby 
<%= f.label :comments do %>
  <%= label_text_for_form ... %>
  <aside></aside>
<% end%>